ansible/roles/oracle-19c/defaults/main.yml

```diff
@@ -35,13 +35,16 @@ grid_install_script: grid_install.sh
 
 password_response_file: grid_pw.rsp
 
-db_ssm_passwords:
+use_ssm_params: false
+db_secretsmanager_passwords:
   asm:
     parameter: "/ec2/{{ ec2_name }}/asm-passwords"
```

Comment:
Unfortunately I named that differently.  It's "secret" for secretsmanager role and "parameter" for ssm parameter role.  So unless you change the ansible roles, you'll need to cut + paste instead.